src/main/scala/mesosphere/marathon/state/AppDefinition.scala

```diff
@@ -247,6 +252,8 @@ case class AppDefinition(
       cpus = resourcesMap.getOrElse(Resource.CPUS, this.cpus),
       mem = resourcesMap.getOrElse(Resource.MEM, this.mem),
       disk = resourcesMap.getOrElse(Resource.DISK, this.disk),
+      // TODO (Yubo): Change hard-coded "gpus" to "Resource.GPUS" after mesos-util has GPU resource type support.
```

Comment:
I submitted the issue here: https://github.com/mesosphere/mesos-utils/issues/27, this is another project need to support "Resource.GPUS" resource type.